sam_mods.c

```diff
@@ -507,15 +518,21 @@ int bam_mods_at_next_pos(const bam1_t *b, hts_base_mod_state *state,
                 if (*cp == ',')
                     break;
             state->MMend[i] = cp;
-            if (cp != state->MM[i])
-                state->MMcount[i] = strtol(cp+1, NULL, 10);
-            else
+            if (cp != state->MM[i]) {
```

Comment:
And now there's no need to modify a bunch of surrounding lines by adding `{ … }`.

Reply:
Removed.